src/raccoon_spotter/utils/data_visualization.py

```diff
@@ -0,0 +1,41 @@
+import cv2
+import numpy as np
+from kedro.config import OmegaConfigLoader
+
+
+def verify_bounding_box(index):
+    catalog_conf = OmegaConfigLoader(conf_source=".")["catalog"]
+    path = catalog_conf["raccoon_data_array"]["filepath"]
+    data_arr = np.load(path, allow_pickle=True)
+    image_array = data_arr["x"][index]
+    bounding_box = data_arr["y"][index]
+    img = cv2.rectangle(
+        image_array,
+        (bounding_box[0], bounding_box[2]),
+        (bounding_box[1], bounding_box[3]),
+        (255, 0, 0),
+        2,
+    )
+    return img
```

Comment:
This is not needed. Nodes would not use this since they can load data through kedro and users do not need this either since during experimentation one has access to kedro context which provides all the functionalities nodes use under the hood. In any case there should never arise a need to load things directly from filepath.